closure/goog/structs/avltree_test.js

```diff
@@ -630,6 +630,45 @@ function testRemoveRightLeftCase() {
 }
 
 
+/**
+ * This test verifies that the copy functionality works correctly.
+ */
+function testCopy() {
+  var tree = new goog.structs.AvlTree((a, b) => a - b);
+  tree.add(100);
+  tree.add(150);
+  tree.add(50);
+  tree.add(0);
+  tree.add(112);
+  tree.add(200);
+  tree.add(125);
+
+  tree.remove(0);
+
+  tree = tree.copy();
+
+  //   100                            100                             112
+  //   / \                            / \                            /  \
+  // 50   150   Right Rotate (150)   50  112      Left Rotate(100)  100  150
+  //     /   \   - - - - - - - ->           \     - - - - - - - ->  /   /   \
+  //    112  200                            150                    50   125  200
+  //     \                                  /  \
+  //     125                               125 200
```

Comment:
Most of the above is copy and pasted from another unit test and isn't really relevant. At the very least delete this comment, we aren't really testing the rotation here.